src/pages/components/WebPopupHandler.js

```diff
@@ -136,9 +136,16 @@ function openContextMenu(data) {
 
         if (_contextMenu) {
             _contextMenu.linkHref = linkHref
-            _contextMenu.linkTitle = linkTitle.trim()
             _contextMenu.imageSrc = imageSrc
+            _contextMenu.linkTitle = linkTitle.trim()
+            _contextMenu.linkProtocol = data.linkProtocol || ""
```

Comment:
Amended a fallback for linkProtocol. Image urls do not specify protocol.

Long press on top of an image, select open to new tab, on the tab new long press on top of the opened image.